scheduler/controllers/nodecollectorsgroup/common.go

```diff
@@ -78,12 +83,23 @@ func getMemorySettings(odigosConfig common.OdigosConfiguration) odigosv1.Collect
 		gomemlimitMiB = nodeCollectorConfig.GoMemLimitMib
 	}
 
+	cpuRequestm := defaultRequestCPUm
+	if nodeCollectorConfig != nil && nodeCollectorConfig.RequestCPUm > 0 {
+		cpuRequestm = nodeCollectorConfig.RequestCPUm
+	}
+	cpuLimitm := defaultLimitCPUm
+	if nodeCollectorConfig != nil && nodeCollectorConfig.LimitCPUm > 0 {
+		cpuLimitm = nodeCollectorConfig.LimitCPUm
+	}
```

Comment:
If a user provides request > limit do we want to check for this here or just let k8s return the error once we deploy?

Reply:
We are not handling it at the moment, can be addressed as a separate PR later on